components/icons/images/tier2/merge-icon.svg

```diff
@@ -0,0 +1,19 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
+  <defs>
+    <clipPath id="clip-_24x24-merge-icon">
+      <rect width="24" height="24"/>
+    </clipPath>
+  </defs>
+  <g id="_24x24-merge-icon" data-name="24x24-merge-icon" clip-path="url(#clip-_24x24-merge-icon)">
+    <rect width="24" height="24" fill="#fff"/>
+    <g id="Group_5080" data-name="Group 5080" transform="translate(-0.215)">
+      <path id="Path_3547" data-name="Path 3547" d="M5.6-1h8.029a2.153,2.153,0,0,1,2.153,2.153v8.893A2.153,2.153,0,0,1,13.633,12.2H3.153A2.153,2.153,0,0,1,1,10.045V5.222" transform="translate(7.215 10)" fill="none" stroke="#565a5c" stroke-linecap="round" stroke-width="2"/>
+      <path id="Path_3548" data-name="Path 3548" d="M13.434-1h.122a2.22,2.22,0,0,1,2.229,2.153v8.893A2.153,2.153,0,0,1,13.633,12.2H2.153A2.153,2.153,0,0,1,0,10.045V9.081" transform="translate(18 14.197) rotate(180)" fill="none" stroke="#565a5c" stroke-linecap="round" stroke-width="2"/>
+      <g id="Group_5073" data-name="Group 5073" transform="translate(-94 -52)">
+        <line id="Line_7" data-name="Line 7" x2="6" y2="6" transform="translate(101.5 60.5)" fill="none" stroke="#565a5c" stroke-linecap="round" stroke-width="2"/>
+        <path id="Path" d="M4.837,2.893,6.161,1.559a.931.931,0,0,1,.65-.267.934.934,0,0,1,.92.92v4.6a.923.923,0,0,1-.92.92h-4.6a.908.908,0,0,1-.846-.57.922.922,0,0,1,.193-1L2.893,4.834l.313-.322Z" transform="translate(103.706 62.707)" fill="#494c4e"/>
+      </g>
+      <path id="Path_3549" data-name="Path 3549" d="M13.434-1h.122a2.22,2.22,0,0,1,2.229,2.153v8.893A2.153,2.153,0,0,1,13.633,12.2H2.153A2.153,2.153,0,0,1,0,10.045V9.081" transform="translate(18 14.197) rotate(180)" fill="none" stroke="#565a5c" stroke-linecap="round" stroke-width="2"/>
+    </g>
+  </g>
+</svg>
```

Comment:
Can you follow the [instructions here](https://github.com/BrightspaceUI/core/tree/master/components/icons#updating-or-contributing-new-icons) which should help clean up a lot of the unnecessary stuff in here (`id`s, `data-*` attributes, etc.).

Also you need to run a build script which will generate some new entries in the catalogue README.

Reply:
Which scripts will I need to run?

Reply:
It's from this bit in the docs:
> The `catalogue.md` catalogue file is automatically generated. When making icon modifications, re-generate this file by running `npm run build:icons`.